frontend/pages/software/ManageSoftwarePage/SoftwareTableConfig.tsx

```diff
@@ -230,13 +230,10 @@ const generateTableHeaders = (isPremiumTier?: boolean): Column[] => {
             <TextCell value={cellProps.cell.value} />
           </span>
           <span className="hosts-cell__link">
```

Comment:
@fleetdm/frontend how do we feel about this pattern of putting the view all hosts link into the last column's cell? Or should view all hosts have its own separate column like on other TableConfigs?

Trying to think which one is easier to maintain/style...